.github/workflows/changelog.yaml

```diff
@@ -10,12 +10,12 @@ permissions:
 
 jobs:
   generate_changelog:
+    name: Generate changelog
+    if: github.repository == 'argoproj/argo-workflows'
```

Comment:
What's the purpose of this change?

Reply:
See my opening comment, which explicitly says:

> * reordering of fields
>     * follow a consistent order of `name` -> `if` -> `needs`
then `uses`, `steps`, or `run` depending on the context